### DISCUSSION TITLE
How to monitor MySQL using Signoz?

### DISCUSSION THREAD

**Discussion by auyongjinyoo** [SigNoz/signoz#3148] [Q&A]
I would like to know, for monitor the database like MySQL. Others than setup own OTEL collector to collect the mysql, is that have any others approach? Example like monitor jvm just need to include the agents in the mysql?

Thanks in advance.

**Comment by vnay92** [comment]
I believe this doc will help: https://signoz.io/blog/opentelemetry-mysql-metrics-monitoring/
